.nvmrc

```diff
@@ -1 +1 @@
-v18.19.0
+v20.19.4
```

Comment:
Is there anything blocking us from bumping to v22? Or is that planned in a future ticket?